packages/client/src/forms/register/fieldMappings/birth/query/registration-mappings.ts

```diff
@@ -245,7 +244,7 @@ export const registrarNameUserTransformer = (
   targetSectionId?: string,
   targetFieldName?: string,
   __?: IOfflineData,
-  userDetails?: IUserDetails
+  userDetails?: User
```

Comment:
UserDetails

Reply:
Should we remove this `userDetails` parameter because it is declared but its value is never read?
It has been used in the functions: `registrarNameUserTransformer, roleUserTransformer and registrarSignatureUserTransformer ` in the file: packages/client/src/forms/register/fieldMappings/birth/query/registration-mappings.ts